content/10.blackbox.md

```diff
@@ -1,17 +1,15 @@
 ## Tip 8: Your DL models can be more transparent {#blackbox}
 
-In ML, interpretability refers to the study of the discriminative features used for classification or regression task.
-ML models can vary in terms of interpretability from a “transparent” to a “black-box” model, the first with a clear description of features importance found, for example, in common random forests implementations [@doi:10.1023/A:1010933404324]. 
-The second for the most widely used DL implementations.  
-Because of the large number of parameters and non-linear relationships among features, DL models are hard to interpret when compared to other ML models. 
+In ML, a model's interpretability refers to the ability to identify the discriminative features used for learning.
+ML models vary in widely in terms of interpretability, ranging from being fully transparent to being a “black-box”.
```

Comment:
> ML models vary in widely in terms of interpretability, ranging from being fully transparent to being a “black-box”.

ML models vary ~~in~~ widely in terms of interpretability, ranging from being fully transparent to being a “black-box**.”**